style/ruby/.rubocop.yml

```diff
@@ -90,6 +90,11 @@ Metrics/AbcSize:
                  branches, and conditions.
   Enabled: false
 
+Metrics/BlockLength:
+  CountComments: false  # count full line comments?
```

Comment:
I vote: `true`.

- We tend not to use comments in our code to begin with.
- This heuristic is, roughly, "is this method easy to quickly understand?".
- Comments indicate that the method is not easy to quickly understand.
- I even believe that comments should incur a 2x penalty, though I doubt this is an option.